docs/atoms/imagery.html

```diff
@@ -1,632 +1,113 @@
+{% include section.html
+  id = "full-width"
+  classes = "border-t-light-xs p-tb-5-xs"
+  title = "Full Width"
+  vocabulary = "Full Width Image"
+  syntax = "Full width photographs span the width of the white section."
+  semantics = "These are used when you want a section of content to stand out as a featured section."
+  html = "imagery/code/full_width.html"
+  css = "imagery/code/full_width.css"
+  example = "imagery/examples/full_width.html"
+%}
 
-      <div class="flex-block-xs flex-wrap-xs border-t-light-xs border-b-light-xs p-tb-2-xs m-tb-4-xs">
-        <h3>Full Width</h3>
-        <div class="flex-basis-100-xs p-t-05-xs p-b-3-xs">
-          <div class="border-light-xs p-3-xs m-t-2-xs round">
-            <h4 class="p-b-3-xs">Atmosphere</h4>
-            <h5 class="p-b-1-xs">Vocabulary</h5>
-            <p class="p-b-2-xs">Full Width Image</p>
-            <h5 class="p-tb-1-xs">Syntax</h5>
-            <p class="p-b-1-xs">Full width photographs span the width of the white section.</p>
-            <h5 class="p-tb-1-xs">Semantics</h5>
-            <p>These are used when you want a section of content to stand out as a featured section.</p>
-          </div>
-        </div>
-        <div class="flex-basis-100-xs flex-basis-100-md flex-basis-50-lg p-r-0-xs p-r-3-md p-r-3-lg code-example">
-          <ul class="tabs clearfix list-unstyled border-b-mid-xs p-t-2-xs text-6-xs" data-tabgroup="image-full">
-            <li><a href="#image-full-html" class="active block-xs text-center-xs p-lr-1-xs p-b-05-xs">HTML</a></li>
-            <li><a href="#image-full-css" class="block-xs text-center-xs p-lr-1-xs p-b-05-xs">CSS</a></li>
-          </ul>
-          <div id="image-full" class="tabgroup">
-            <div id="image-full-html">
-              <div class="guide-code m-b-3-xs">
-                <pre><code class="language-html">&lt;td class="text-center-xs image-full" align="center"&gt;
-          &lt;img width="600" src="image-full.png" alt="Placeholder"&gt;
-      &lt;/td&gt;</code></pre>
-              </div>
-            </div>
-            <div id="image-full-css">
-              <div class="guide-code m-b-3-xs">
-                <pre><code class="language-css">.image-full img {
-        width: 600px;
-        min-width: 100%;
-        max-width: 100%;
-        height: auto;
-        border: 0;
-        line-height: 100%;
-        outline: none;
-        text-decoration: none;
-        text-align: center;
-  }</code></pre>
-              </div>
-            </div>
-          </div>
-        </div>
-        <div class="flex-basis-100-xs flex-basis-100-md flex-basis-50-lg p-3-xs bg-gray-light email-example">
-          <div class="text-bg-white p-tb-5-xs h-full-xs round">
-            <img class="w-full-xs" src="{{ site.baseurl }}/img/image-full.png" alt="Placeholder">
-          </div>
-        </div>
-      </div>
+{% include section.html
+  id = "inline"
+  classes = "border-t-light-xs p-tb-5-xs"
```

Comment:
@hopearmstrong We can pass css classes the the section.html include here.  These might need to be adjusted.